### PR TITLE
Fixing celavi-data issue #83

### PR DESCRIPTION
**Pull request by jwalzberg**
Solves issue #242

### CODE REVIEW

**Comment by rjhanes** [comment]
One change to make before merging: The `solar_glass_recovery` method is expecting a solar glass recovery entry under the path_split dict in scenario.yaml. I think this material loss is covered with the value given for module disassembly (the processing step which precedes solar glass recovery). To complete testing I commented out the loss parameter lines in `solar_glass_recovery`. and set _loss = 0 in my local code. ~~Those lines should be removed or the loss parameter added to scenario.yaml before merging.~~ I'll take care of the necessary changes and merging

One suggested change (can happen now or later): The docstrings for new methods aren't always up to date, and are a good place to document data sources and model assumptions for cost methods.

No change needed, just a note that the single set of cullet manufacturing parameters was being used in both solar and window glass cullet manuf by design: those are identical technologies, just separated so the two glasses don't mix in the model.

**Review by rjhanes — CHANGES_REQUESTED** [review]
I forgot how PR reviews work and just left my review with one suggested change and one optional change in a separate comment.
